tests/console/validate_zfcp.pm

```diff
@@ -0,0 +1,199 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Validate zfcp in the installed system:
+#          - Verification of FCP devices online
+#          - Verificaiton of FCP devices
+#          - Verification of LUNs visible as SCSI devices
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use scheduler 'get_test_suite_data';
+use utils qw(arrays_subset);
```

Comment:
minor `use utils 'arrays_subset';`

Reply:
leave it as it is. no problem. FYI it does not break any policy. qw is just syntax sugar of the list of args